test/test_posv.cc

```diff
@@ -82,7 +82,7 @@ void test_posv_work(Params& params, bool run)
         params.time2.name( "potrf (s)" );
         params.time3.name( "potrs (s)" );
     }
-    else if (timer_level >=2 && (params.routine == "posv_mixed")) {
+    else if (timer_level >=2 && (params.routine == "posv_mixed" || params.routine == "posv_mixed_gmres")) {
```

Comment:
Have you ever thought about implementing `cpplint` to check code formatting? There are some checks when you run `git commit`, but `cpplint` you could run before committing. 

But the 80 character wrap is pretty universal I know it was the same with Python. So I should of kept that in mind. Thanks for pointing that out.

Reply:
I haven't looked at cpplint, that's a good idea.

There is a script `tools/slate-style-reject.pl` that runs on commit. It doesn't currently check for 80 chars (or 85, the supposed "hard limit"), I think because there were enough exceptions for it to be annoying to block commits: 2131 (or 1049) of 136440 lines in include, src, test.